src/packages/wallet/chains.ts

```diff
@@ -7,7 +7,7 @@ const getSupportedChains = (): Chain[] => {
     return [bscTestnet, opBNBTestnet, sepolia];
   }
 
-  return [bsc, mainnet];
+  return [bsc, mainnet, opBNB];
```

Comment:
For now I suggest only enabling opBNB on the preview environment, otherwise if we merge this PR then deploy we'll enable it in production

Reply:
The goal is to get this to production soon, but we'll keep this as a draft until then. Only the bridge has been activated so far, we'll wait for the rest of the things to get added.

Reply:
Ok got you, thanks for the info.